frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -183,3 +185,17 @@ func (h *createUploadDirectoryHandler) ServeHTTP(w http.ResponseWriter, r *http.
 	}
 	operator.ReplyJSONOK(w, res)
 }
+
+type listUploadDirectoriesHandler struct {
+	m UserArtifactsManager
+}
+
+func (h *listUploadDirectoriesHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	res, err := h.m.ListDirs()
+	if err != nil {
+		log.Printf("error retrieving upload tokens: %+v", err)
```

Comment:
This is the only place with logging in this file, is that intentional?

Reply:
good point, removed it from here.